.github/workflows/tests.yml

```diff
@@ -28,4 +28,4 @@ jobs:
           pip install -r requirements.txt
       - name: Run tests with coverage
         run: |
-          pytest -k "(unit or test_basic_commands) and not slow" --cov --cov-report term-missing
+          pytest -k "not slow" --ignore=tests/shell/test_debug_configs.py --ignore=tests/shell/test_sweeps.py --cov --cov-report term-missing
```

Comment:
Why are these two files ignored?

Reply:
Disclaimer: It is a bit off that this change is also in this PR, but I noticed the problem with the original code during implementing the test for the json serialize.

So, the initial idea was to only test the unit tests and the basic shell tests. However, the unit tests were not executed in any why with the previous statement, this change fixes this. The reason for only having these tests for github CI is that nearly all of them take a very long time (they are about sweeps etc) and they are more or less meant as starting points to create your own tests. For sure, this is debateable, but maybe we discuss this in a separate issue/PR, or what do you think?